src/Provider/Provider.Web/Configuration/ConfigurationExtensions.cs

```diff
@@ -72,7 +72,7 @@ public static void AddMvcService(this IServiceCollection services, IHostingEnvir
 
                 opts.AddTrimModelBinderProvider(loggerFactory);
             })
-            .AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<Startup>())
+            .AddFluentValidation()
```

Comment:
Same concern as above?

Reply:
Have just rechecked and it seems fine.  The issue we had previously was I'd removed `AddFluentValidation` entirely. Now its still calling `AddFluentValidation` and we need to explicitly register the validators

Reply:
👍